openjdk.build/playlist.xml

```diff
@@ -1,3 +1,16 @@
+<!--
```

Comment:
You can add this line: \<?xml version="1.0" encoding="UTF-8"?\> before copyright comment.

Reply:
Thanks. Added.